### PR TITLE
New package: gtk-sharp2-2.12.45

### PR DESCRIPTION
**Pull request by Johnnynator**

### CODE REVIEW

File: srcpkgs/gtk-sharp2/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+version=2.12.45
+revision=1
+build_style=gnu-configure
+wrksrc="gtk-sharp-${version}"
+configure_args="--disable-static"
+hostmakedepends="pkg-config mono"
+makedepends="mono-devel gtk+-devel libglade-devel"
+maintainer="John <johnz@posteo.net>"
+homepage="http://www.mono-project.com/docs/gui/gtksharp/"
+license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/gtk-sharp2/template]: use spdx identifier

File: srcpkgs/gtk-sharp2/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+pkgname=gtk-sharp2
+version=2.12.45
+revision=1
+build_style=gnu-configure
+wrksrc="gtk-sharp-${version}"
+configure_args="--disable-static"
+hostmakedepends="pkg-config mono"
+makedepends="mono-devel gtk+-devel libglade-devel"
+maintainer="John <johnz@posteo.net>"
+homepage="http://www.mono-project.com/docs/gui/gtksharp/"
```
Comment (maxice8) [inline srcpkgs/gtk-sharp2/template]: homepage above distfiles=

File: srcpkgs/gtk-sharp2/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+license="LGPL-2.1"
+short_desc="Graphical User Interface Toolkit for mono and .Net (Gtk#)"
```
Comment (maxice8) [inline srcpkgs/gtk-sharp2/template]: short_desc above license=

File: srcpkgs/gtk-sharp2/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'gtk-sharp2'
+pkgname=gtk-sharp2
+version=2.12.45
+revision=1
+build_style=gnu-configure
+wrksrc="gtk-sharp-${version}"
```
Comment (maxice8) [inline srcpkgs/gtk-sharp2/template]: wrksrc under reivision=

**Comment by Johnnynator** [comment]
Ok. Fixed them.

**Pull request merged**